pom.xml

```diff
@@ -12,6 +12,8 @@
 	<properties>
         <project.build.sourceEncoding>8859_1</project.build.sourceEncoding>
         <java.version>1.8</java.version>
+		<swt.version>4.6.1</swt.version> <!-- build uses libs in uis/libs/*.jar -->
```

Comment:
I found that the official 4.9 jar file contains a `version.txt` stating version `4.919`. 
My theory is that the one contained in the `uis/lib` folder is a dev version.

I'll change it to use the official 4.9 version.

Reply:
For building purposes, latest release build of SWT is fine.  We often ship a nightly build of SWT because they fix specific issues we've run into.  We also have a custom SWT for Mac with about 5 lines of code changes.  But that's not important for building :)

Reply:
One challenge is finding a repository holding the latest SWT jars.
I made a fork of one I discovered. Maybe BiglySoftware would be a better place for that fork?

Reply:
I discovered that the official 4.10 version states "4.922" in its `version.txt`.
If anyone has Linux or is able to run the bash `newRelease.sh` script then feel free to update the repo with version 4.10 and soon to come version 4.11